app/controllers/admin/student_accounts_controller.rb

```diff
@@ -0,0 +1,37 @@
+require 'date'
```

Comment:
I don't think that you need to do this. I believe that `'date'` is present/required by default in Ruby/Rails.

Reply:
This change has been made.